src/index.html

```diff
@@ -8,6 +8,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="rate stars--0">
+      <div class="rate__star"></div>
+      <div class="rate__star"></div>
+      <div class="rate__star"></div>
+      <div class="rate__star"></div>
+      <div class="rate__star"></div>
+    </div>
+    <div class="rate stars--1">
```

Comment:
```suggestion
    </div>

    <div class="rate stars--1">
```
add some empty lines

Reply:
added